client/app/bundles/Zoon/components/WorkflowDiagram.jsx

```diff
@@ -1,20 +1,28 @@
 import React from "react"
 import Reorder from "react-reorder"
+import classnames from "classnames"
+
+import {upFirstLetter} from "../utils"
 
 const families = ["occurrence", "covariate", "process", "model", "output"]
+export const allCompositionTypes = ["list", "chain", "replicate"]
 
-const Switch = ({ id, checked, onChange }) => (
-  <span className="switch tiny" style={{"margin": "0em 0.5em"}}>
-    <input
-      type="checkbox"
-      className="switch-input"
-      id={id}
-      checked={checked}
-      onChange={onChange}
-    />
-    <label className="switch-paddle" htmlFor={id} />
-  </span>
-)
+const Switch = ({ value, onChange }) => <div className="module-list__switch">
+  {allCompositionTypes.map((compType) => {
+    const isSelected = value === compType
+    return <span key={compType} onClick={() => onChange(compType)}>
+      <i
+        className={classnames(
+          "fa",
+          isSelected ? "fa-dot-circle-o" : "fa-circle-o",
```

Comment:
I've thought about that, but I preferred the ternary since it clearly communicates it's a mutually exclusive choice. It does look a little clunkier.

Pretty sway-able on this one. Which one do you prefer?